### ISSUE TITLE
Improve testing during CI/CD

### ISSUE THREAD

**Issue by tbeck** [ocupop/librestream#120]
**Is your feature request related to a problem? Please describe.**
There are inadvertent side effects of code changes that are not being caught through manual checks. ie... missing images and broken links.

**Describe the solution you'd like**
Create a CI/CD workflow for checking links and image loads on specific pages (list provided)

**Comment by tbeck** [comment]
@samyue888 - adding the list you provided here for reference.
[Items for Ocupop to check.pdf](https://github.com/ocupop/librestream/files/6442359/Items.for.Ocupop.to.check.pdf)

**Comment by samyue888** [comment]
@paulstroot Could we get an update on this? Are all the checks in place?

**Comment by paulstroot** [comment]
@samyue888 , Yes, this should be working. We just wanted to give it a few more tests and monitor how it's working before claiming that all is working as it should be. The Automated tests should be running every time code is pushed to the main branch (it can also be run manually from github). I can let you know here though how it is working.

If you go to the [Librestream Github page](https://github.com/ocupop/librestream), click on "[Actions](https://github.com/ocupop/librestream/actions)" and you will see all of the past tests that have been performed. When you complete a pull-request to merge from develop into main, a new action will appear on that page with the results of the automated test. It might take 5+ minutes to complete. If it fails, let us know and we will investigate. Additionally, every time we open a pull request, we can run the automated test against the develop branch to make sure it passes the tests first.

If this is confusing, we can demonstrate next week on our call.

**Comment by samyue888** [comment]
@paulstroot Could you provide more guidance on how to troubleshoot from the report? For example, right now we're getting this build error for the training page but we cannot pinpoint what exactly is breaking:
![image](https://user-images.githubusercontent.com/81642726/125819344-33df11ae-7835-44fd-93ec-97bc5f2371d5.png)

**Comment by paulstroot** [comment]
Those error reports are difficult to understand. I took a look and the
important part is here:

✓ >>> Visit /contact-us-support/ (436ms)
392
<https://github.com/ocupop/librestream/runs/3069664251?check_suite_focus=true#step:3:392>
✓ H1 tag does not contain the text '404' (783ms)
393
<https://github.com/ocupop/librestream/runs/3069664251?check_suite_focus=true#step:3:393>
1)
<form> tag exists
394
<https://github.com/ocupop/librestream/runs/3069664251?check_suite_focus=true#step:3:394>
2)
Submit button exists

On the page librestream.com/contact-us-support, we are testing to make sure
that the form is properly showing up on the page (by checking that a "form"
tag exists and a "submit" button exists). The problem though is that we
removed that form from the page because we are still having issues with the
recaptcha. Should I remove the test on this page? or is there some sort of
conditional statement we can make to say "only check if a form exists if
some other requirement is met?"

**Comment by samyue888** [comment]
We can keep the test, as long as we know what the issue is.
